### PR TITLE
Use Alpine Linux

### PR DESCRIPTION
**Pull request by danielbayerlein**
Switch to Alpine Linux (https://alpinelinux.org) it saves a lot of space. 👾

### CODE REVIEW

**Comment by danielbayerlein** [comment]
Once https://github.com/yarnpkg/yarn/issues/1326 is done, the use of Yarn will be easier.

**Comment by chrishelgert** [comment]
@danielbayerlein  let´s review this tomorrow together 👍

File: packages/generator-honeycomb/app/templates/_Dockerfile [hunk 1]
```diff
@@ -1,28 +1,33 @@
...
+RUN apk add --no-cache --virtual .app-deps python make g++ \
+  && yarn --pure-lockfile \
+  && yarn cache clean \
+  && apk del .app-deps
 
 # Bundle app source
 COPY . /code/<%= dir %>
 
 # Build
 RUN yarn run build
```
Comment (chrishelgert) [inline packages/generator-honeycomb/app/templates/_Dockerfile]: Option 1:
build dist on local machine and only add the builded sources and production node_modules (no devDependencies - NODE_ENV production)

Option 2:
build dist-files on docker-container, clear node_modules and install only production-dependencies again

Comment (danielbayerlein) [inline packages/generator-honeycomb/app/templates/_Dockerfile]: @chrishelgert At the moment I think option 2 is okay, but it makes the image 3 times bigger. 😳

Comment (danielbayerlein) [inline packages/generator-honeycomb/app/templates/_Dockerfile]: @chrishelgert I'll switch to option 1.

Comment (chrishelgert) [inline packages/generator-honeycomb/app/templates/_Dockerfile]: @danielbayerlein nice choice 👍

**Comment by danielbayerlein** [comment]
@chrishelgert Ready to review 👨‍💻

File: packages/generator-honeycomb/app/templates/_Dockerfile [hunk 2]
```diff
@@ -1,28 +1,30 @@
...
+  && yarn --pure-lockfile --production \
+  && yarn cache clean \
+  && apk del .app-deps
 
 # Bundle app source
 COPY . /code/<%= dir %>
 
-# Build
-RUN yarn run build
+# Set environment variable
```
Comment (chrishelgert) [inline packages/generator-honeycomb/app/templates/_Dockerfile]: if we set the to NODE_ENV to prodution before yarn, then we won´t need --production or does yarn don´t check for ENV?

Comment (chrishelgert) [inline packages/generator-honeycomb/app/templates/_Dockerfile]: https://yarnpkg.com/en/docs/cli/install#toc-yarn-install-production

> Using the --production flag, or when the NODE_ENV environment variable is set to production, Yarn will not install any package listed in devDependencies.

Comment (danielbayerlein) [inline packages/generator-honeycomb/app/templates/_Dockerfile]: Done 😉

**Comment by chrishelgert** [comment]
nice on 👍

**Pull request merged**